packages/stream_chat_flutter/CHANGELOG.md

```diff
@@ -6,27 +6,34 @@
   Consonants as Emoji.
 - [#1505](https://github.com/GetStream/stream-chat-flutter/issues/1505) Fixed Message bubble disappears for Hangul
   Consonants.
+- [[#1476]](https://github.com/GetStream/stream-chat-flutter/issues/1476) Fixed `UserAvatarTransform.userAvatarBuilder`
```

Comment:
Is there any purpose for double square brackets? 🤔

Reply:
It renders with bracket.

Eg: [#1426]